diff --git a/target/linux/ramips/dts/mt7621_netgear_r6220.dts b/target/linux/ramips/dts/mt7621_netgear_r6220.dts
index cc29d4ea8e..3e0b7d4672 100644
--- a/target/linux/ramips/dts/mt7621_netgear_r6220.dts
+++ b/target/linux/ramips/dts/mt7621_netgear_r6220.dts
@@ -37,9 +37,9 @@
 			reg = <0x600000 0x1c00000>;
 		};
 
-		factory: partition@2e00000 {
+		factory: partition@2de0000 {
 			label = "factory";
-			reg = <0x2e00000 0x100000>;
+			reg = <0x2de0000 0x100000>;
 			read-only;
 		};
 
